Modules/Sources/Storage/GRDB/Model/PersistedProduct.swift

```diff
@@ -99,11 +103,20 @@ extension PersistedProduct: FetchableRecord, PersistableRecord {
 // MARK: - Point of Sale Requests
 public extension PersistedProduct {
     /// Returns a request for POS-supported products (simple and variable, non-downloadable) for a given site, ordered by name
+    /// Filters out products with trash, draft, pending, or private status to ensure only published and 3rd party custom status products are shown
     static func posProductsRequest(siteID: Int64) -> QueryInterfaceRequest<PersistedProduct> {
+        let excludedStatuses = [
```

Comment:
Would make sense to import Networking here and use the raw values for `ProductStatus` rather than the string directly?